src/open_inwoner/utils/ckeditor.py

```diff
@@ -105,3 +118,18 @@ def get_product_rendered_content(product):
                 element.append(icon)
 
     return soup
+
+
+def convert_styling_to_nlds(response: requests.Response) -> None:
+    """
+    Post-process rendered HTML content, replace legacy styling with NLDS classes
+    """
```

Comment:
Superfluous comment feels dogmatic: it just duplicates information already present in the function signature.